### DISCUSSION TITLE
Option to open Charmstone when the modifier keys are pressed without mouse movement.

### DISCUSSION THREAD

**Discussion by sculliga** [rxhanson/Charmstone-Community#49] [Ideas]
At the moment, it seems the only way to get Charmstone to open is to press the modifier keys and start moving the mouse for the icons to appear.

I would prefer an option if when pressing the modifier keys, the icons could just show up without moving the mouse.

**Comment by rxhanson** [comment]
Move the "Min Dist to Appear" slider all the way to the left. Let me know if that's not what you're looking for.

**Reply by sculliga** [reply]
That worked perfectly! Thank you!
